metrics/network/remote_network/README.md

```diff
@@ -0,0 +1,22 @@
+# Clear Containers remote networking test
+
+- `remote-networking-iperf3.sh`: measures bandwidth using iperf3 using a remote setup. Host A will run 
+a container that will act as a server while host B will run a container that will act as a client 
+and the network bandwidth will be measure across the containers.
+
+## Prerequisite
+An automatic login from host A / user a to host B /user b is needed. It is necessary to setup authentication 
+keys in order to do an ssh login without password.
+
+## Running the remote networking test
+
+`remote-networking-iperf3.sh` should run in host A. The script needs three inputs to run: interface name 
```

Comment:
Can we change this to have bullets to make it clearer? something like: 

- interface name where swarm will run
- the user
- the IP address of the Host B